### PR TITLE
Bitwarden CLI tool definition

### PR DESCRIPTION
**Pull request by atjbramley**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Feature update - support and definitions for Bitwarden CLI tool
(https://bitwarden.com/help/cli/)

**Additional info:**

Thanks for the help with the spec - sorry this took so long!

### CODE REVIEW

**Comment by atjbramley** [comment]
👍

**Comment by atjbramley** [comment]
Sorry - I've realised that 3 more files have been included in this along with the bw.ts - just stuff I used for testing 🤦

Luckily nothing important - please get rid of these obviously!

File: src/bw.ts [hunk 1]
```diff
@@ -0,0 +1,1555 @@
+// Autogenerated by @fig/complete-commander
+
+import { filepaths } from "@fig/autocomplete-generators";
+import { Server } from "http";
+import { collapseTextChangeRangesAcrossMultipleVersions } from "typescript";
```
Comment (mschrage) [inline src/bw.ts:5]:
```suggestion
```

File: src/bw.ts [hunk 2]
```diff
@@ -0,0 +1,1555 @@
...
+            "--length",
+          ],
+          priority: 49,
+        },
+      ],
+    },
+    {
+      name: "encode",
+      description: "Base 64 encode stdin",
+      icon: "https://bitwarden.com/images/icon_32x32.png",
```
Comment (mschrage) [inline src/bw.ts:409]: Can you pull this icon URL into a constant? 
```suggestion
      icon: "https://bitwarden.com/images/icon_32x32.png",
```

**Review by mschrage — CHANGES_REQUESTED** [review]
This look great! Can you remove the extraneous files from the PR and make the changes below, so we can get this merged in?

**Review by mschrage — APPROVED** [review]

**Comment by fedeci** [comment]
recheck

**Comment by atjbramley** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by fedeci** [comment]
recheck

**Pull request merged**